lib/accessly/permitted_actions/on_object_query.rb

```diff
@@ -31,6 +31,38 @@ def can?(action_id, object_type, object_id)
             ).exists?
         end
       end
+
+      # Returns an ActiveRecord::Relation of object_type containing the
+      # records on which the actor has permission to perform action_id.
+      #
+      # @param action_id [Integer] The action we're checking on the actor for the object.
+      # @param object_type [ActiveRecord::Base] The ActiveRecord model to be loaded.
+      # @raise [Accessly::ListError] if the object_type is not of type ActiveRecord::Base
+      # @return [ActiveRecord::Relation]
+      #
+      # @example
```

Comment:
Instead of comment on each of the examples... I _think_ (I'm still learning 😆) that the way you would do this in yard is to have separate example tags and have the title of the example next to it like: `@example Give me the list of Posts on which the user has permission to perform action_id 3` and `@example Give me the list of Posts on which the user has permission to perform action_id 3 on segment 1`? [Here is where I saw that in the docs before.](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#example) does it look that way to you too?

Reply:
Makes sense to me - Do you think that is a change we could make broadly to all the other files (like the Integer change) after this gets merged?

Reply:
Yeah... maybe as we touch them again? 🤷‍♂️

Reply:
https://app.clubhouse.io/lessonly/story/9527/break-out-example-comments-yard-doc